libcaja-private/caja-clipboard.c

```diff
@@ -323,8 +331,10 @@ merge_in_clipboard_menu_items (GObject *widget_as_object,
 
     add_selection_callback = target_data->shares_selection_changes;
 
+    G_GNUC_BEGIN_IGNORE_DEPRECATIONS
     gtk_ui_manager_insert_action_group (target_data->ui_manager,
```

Comment:
Do we want to silence this one?

Reply:
Done Silencer is removed.
But i am not sure if GtkUIManager can migrate to GtkBuilder at this place with using GtkAction.